Library/Homebrew/github_packages.rb

```diff
@@ -101,6 +101,18 @@ def self.root_url(org, repo, prefix = URL_PREFIX)
     "#{prefix}#{org}/#{repo_without_prefix(repo)}"
   end
 
+  def self.sanitize_root_url(url)
```

Comment:
I’d just make this part of root_url unless there’s cases where it shouldn’t be used (in which case: what are they)?